build-pipelines/buildpacks/tasks/build_release.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+function build_release() {
+  registry="${1}"
+  organization="${2}"
+  username="${3}"
+  password="${4}"
+  stemcell_image="${5}"
+  release_name="${6}"
+  release_url="${7}"
+  release_version="${8}"
+  release_sha1="${9}"
+
+  echo -e "Release information:"
+  echo -e "  - Release name:    ${GREEN}${release_name}${NC}"
+  echo -e "  - Release version: ${GREEN}${release_version}${NC}"
+  echo -e "  - Release URL:     ${GREEN}${release_url}${NC}"
+  echo -e "  - Release SHA1:    ${GREEN}${release_sha1}${NC}"
+
+  build_args=(
+    --stemcell="${stemcell_image}"
+    --name="${release_name}"
+    --version="${release_version}"
+    --url="${release_url}"
+    --sha1="${release_sha1}"
+    --docker-registry="${registry}"
+    --docker-organization="${organization}"
+  )
+
+  built_image=$(fissile build release-images --dry-run "${build_args[@]}" | cut -d' ' -f3)
+  built_image_tag="${built_image#*:}"
+  creds_string=""${username}":"${password}""
```

Comment:
This doesn't seem to be used anymore. Could you run shellcheck on those shell files?